app/routes/docs.$lang.$ref.$.tsx

```diff
@@ -29,8 +29,14 @@ export async function loader({ params, request }: LoaderFunctionArgs) {
   let url = new URL(request.url);
   let pageUrl = url.protocol + "//" + url.host + url.pathname;
   invariant(params.ref, "expected `ref` params");
+  let splat = params["*"];
+  let isChangelog = splat?.endsWith("/changelog");
   try {
-    let doc = await getRepoDoc(params.ref, params["*"] || "index");
+    let doc = await getRepoDoc(
+      params.ref,
+      isChangelog ? "CHANGELOG" : params["*"] || "index",
+      isChangelog ? "" : "docs",
```

Comment:
this is the area in particular that could probably be improved..

Reply:
If you go with my other comment, it can simplify it a bit, but you still have still have the issue of hardcoding a "magic" route

Personally I'm pretty okay with this for a one off, and if there are more like this we can start figuring out a better system for it.

My thought at first was to create a unique route for it, but I'm realizing that's a lot more setup for what you've already accomplished.